e2e/tests/omnibus-base-sepolia-andromeda.toml/Collateral_Limits.e2e.js

```diff
@@ -14,7 +14,7 @@ const { wrapCollateral } = require('../../tasks/wrapCollateral');
 const { unwrapCollateral } = require('../../tasks/unwrapCollateral');
 const { syncTime } = require('../../tasks/syncTime');
 
-const SYNTH_USDC_MAX_MARKET_COLLATERAL = 10_000_000;
+const SYNTH_USDC_MAX_MARKET_COLLATERAL = 20_000_000;
```

Comment:
When people managed to fill up 10mil wrapper? I thought we did not have free mint to avoid this...